source/IntegrationConfigFactory.ts

```diff
@@ -15,20 +14,20 @@ import { IntegrationType } from 'aws-sdk/clients/apigateway';
  */
 @injectable()
 export default class IntegrationConfigFactory {
-    private _logger: Winston.Logger;
-
     /**
      * Creates specifically typed messages given a json string
      * @param {Container} container The IOC container used to resolve other dependencies
      * @memberof MessageHandlerFactory
      */
-    constructor(@inject(TYPES.Logger) logger: Winston.Logger) {
-        this._logger = logger;
+    constructor(
+        @inject(TYPES.DDLHelper)
+        @named(IntegrationSystemType.Oracle)
+        private readonly oracleDDLHelper: IDDLHelper) {
     }
 
     public create(integrationType: IntegrationType): IIntegrationConfig {
         switch (integrationType) {
-            case 'Banner': {
+            case IntegrationType.Banner: {
```

Comment:
Now that we have an ingestion service, it won't be too hard to consider moving this to the service side.